### PR TITLE
Pt family ui ingress

### PR DESCRIPTION
**Pull request by stefancocora**

### CODE REVIEW

File: scripts/deploy_kd [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+archiveGitCommit
+getKd
+getKubectl
+
+export PATH=$PWD:$PATH
+
+kubectl version --client
+
+echo "deploying ..."
+export VERSION=${VERSION}
```
Comment (vaijab) [inline scripts/deploy_kd]: You can just do: `export VERSION`.

File: scripts/deploy_kd [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+KD_DOWLOAD_ADDR="https://github.com/UKHomeOffice/kd/releases/download/${KD_VERSION}/kd_linux_amd64"
+KUBECTL_BINARY='kubectl'
+KUBECTL_VERSION='v1.3.5'
+KUBECTL_MD5='26852c9bd750a2afee527658944397d1'
+KUBECTL_DOWNLOAD_ADDR="https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl"
+ENVIRONMENT='dev'
+APP='pttg-ip-fm-ui'
+
+function getKd() {
+echo "checking if the desired kd version exists"
```
Comment (vaijab) [inline scripts/deploy_kd]: Could use same indentation as below to make the script more readable.

Comment (stefancocora) [reply scripts/deploy_kd]: I thought I did have indentation, not sure how it got reverted ... 
I'll indent those 2 functions.

**Comment by stefancocora** [comment]
Updated vars and indentation.

**Comment by vaijab** [comment]
LGTM

**Pull request merged**